util-images/probes/OWNERS

```diff
@@ -1,8 +1,10 @@
 approvers:
+- mborsz
```

Comment:
With all your contributions and detailed reviews, I actually suggest that we add you to the top-level owners in this repo.

I would be willing to approve this change, but I would like to give time for @mm4tt and @shyamjvs to objects.

But in general - please open such PR and we will let it wait for a week or so (unless everyone approve by then).

Reply:
Thanks Wojtek, I opened https://github.com/kubernetes/perf-tests/pull/1820